lib/FormCustomProperties/CustomPropertiesListField.js

```diff
@@ -141,11 +141,19 @@ export default class CustomPropertiesListField extends React.Component {
       const { note, publicNote, value } = val ? val[0] : {};
 
       if (!customProperty.primary && customProperty.value && !value) {
-        errors[customProperty.value] = <FormattedMessage id="stripes-core.label.missingRequiredField" />;
+        if (customProperty.type === CUSTOM_PROPERTY_TYPE_NUMBER || customProperty.type === CUSTOM_PROPERTY_TYPE_DECIMAL) {
```

Comment:
While we're doing this, can you change this and any other references to these constants to pull from `customPropertyTypes.js` instead?